src/app/InitializingScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import {
+  BasicLoginCardLayout,
+  BasicLoginPageLayout,
+  Icon,
+  LoginCard,
+  LoginCardHeader,
+  LoginPageContainer,
+  LoginPageFooter,
+  LoginPageHeader,
+  Spinner
+} from 'patternfly-react';
+import { isKioskMode } from '../utils/SearchParamUtils';
+
+const kialiTitle = require('../assets/img/logo-login.svg');
+
+const InitializingScreen: React.FC<{ errorMsg?: string }> = (props: { errorMsg?: string }) => {
```

Comment:
It took me a minute to realize what the difference between React.FC<> and React.SFC<>. Looks like SFC is now deprecated in favor of FC: https://twitter.com/dan_abramov/status/1057625147216220162?lang=en